examples/v2/config_chat2.nim

```diff
@@ -270,24 +270,25 @@ type
       name: "rln-relay-id-commitment" }: string
   
     rlnRelayEthAccount* {.
-      desc: "Ethereum account address for an Ethereum testnet", 
+      desc: "Ethereum account address for an Ethereum testnet",
+      # NOTE: This can be derived from the private key, but kept for future use
       defaultValue: ""
-      name: "eth-account-address" }: string
+      name: "rln-relay-eth-account-address" }: string
     
     rlnRelayEthAccountPrivKey* {.
       desc: "Account private key for an Ethereum testnet",
       defaultValue: ""
-      name: "eth-account-privatekey" }: string
+      name: "rln-relay-eth-account-privatekey" }: string
```

Comment:
```suggestion
      name: "rln-relay-eth-account-private-key" }: string
```
(Throughout most of the codebase "private key" is considered two words, though I'm sure it's up for debate :) )

Reply:
Addressed in [0395e29](https://github.com/status-im/nwaku/pull/1204/commits/0395e29b0706bca9d58705bea5e6e41e6f27afaf)